### ISSUE TITLE
Lost Rewards from Master Weekly Kit

### ISSUE THREAD

**Issue by EpicButler** [ObliqueNET/Server#1103]
I logged onto the server shortly after the 8.2 announcement was made (so about 2 hours ago), and I redeemed my masterweekly kit. I placed all of the items I received from the kit in my shop chests. When the server came back online, I checked my chests, but the items were nowhere to be found.

I tried to enter the command again (in the scenario that the rollback meant that I had never received the kit) but I had to wait 6 days and 22 hours to acquire the kit again.
<img width="956" alt="Screen Shot 2021-05-12 at 11 48 20 PM" src="https://user-images.githubusercontent.com/84116412/118078819-8b574100-b37c-11eb-9015-c42b95bbc684.png">

<img width="1440" alt="Screen Shot 2021-05-12 at 11 44 50 PM" src="https://user-images.githubusercontent.com/84116412/118078547-0e2bcc00-b37c-11eb-9a77-ace2281b15ab.png">
Here's my rare candy chest from after the rollback. I had 24 rare candies (since I added the 16 I got from the kit), but now there's only 8.

**Comment by Rasgnarok** [comment]
Can I have your in-game name, and can you confirm you therefore need 16 rare candies refunded?

**Comment by EpicButler** [comment]
My IGN is EpicButler, and the most "confirming" evidence that I have are the /lg logs:
<img width="1139" alt="Screen Shot 2021-05-14 at 5 23 47 PM" src="https://user-images.githubusercontent.com/84116412/118337316-25cc9700-b4d9-11eb-821c-12f301b3a2bc.png">
So the logs show that I added 16 rare candies at about 10:40 CST, which is before Hiroku's announcement that the server would roll back. My logs show respectively that I added the 16 (making it 24 in the chest), removing them all, adding 1 back in, then adding 15 more, then finally adding the rest. There are no removals at all, so there should be 24 in the chest, and nobody had purchased any.

<img width="1141" alt="Screen Shot 2021-05-14 at 5 24 59 PM" src="https://user-images.githubusercontent.com/84116412/118337394-51e81800-b4d9-11eb-985f-1b6c6b690f57.png">

This is what my chest looks like now (with logs in the background). Again, no purchases have been made, and I still am short 16 rare candies.
